gulp/tasks/scripts.js

```diff
@@ -202,17 +201,39 @@ function scriptsApps() {
   // Run each application's JS through webpack and store the gulp streams.
   const streams = [];
   apps.forEach( app => {
-    streams.push(
-      _processScript(
-        webpackConfig.appsConf,
-        `/apps/${ app }/js/**/*.js`,
-        `/apps/${ app }/js`
-      )
-    );
+    /* Check if node_modules directory exists in a particular app's folder.
+       If it doesn't log the command to add it and don't process the scripts. */
+    // eslint-disable-next-line no-sync
+    if ( fs.existsSync( `${ paths.unprocessed }/apps/${ app }/package.json` ) ) {
```

Comment:
You could probably make `${ paths.unprocessed }/apps/${ app }/` a variable.

Reply:
Good point, added! https://github.com/cfpb/cfgov-refresh/pull/3820/commits/a16e023d99be7fd3a065e78203ae8e58c58a672c